### ISSUE TITLE
Clothing Resizing / Dissapearing on Apply

### ISSUE THREAD

**Issue by Nosuuddo** [artieficial/ApplyAccessories#13]
Using socks on RexEvoloved and whenever I hit apply changes, both resuing and not reusing bones grants me issues.
Pictures Below:
What it is supposed to look like:
![sockissue1](https://user-images.githubusercontent.com/85851448/167355615-f4266c91-ebb3-40f1-b6bc-bb310f5d18a9.PNG)

Re-using bones resizes them:
![sockissuereusebone](https://user-images.githubusercontent.com/85851448/167355692-11f64936-321f-4c9b-ab68-ce382e4ab743.PNG)

Not Re-using bones makes only legs appear from a distance and dissapear when zooming in, arms gone, and both arms/legs boxes are resized:
![sockissuedontreusebonelegs](https://user-images.githubusercontent.com/85851448/167355839-6166f6c6-ccd5-4c94-b690-d80324efc7ff.PNG)
![sockissuedontreuseboneszoomin](https://user-images.githubusercontent.com/85851448/167355843-2504d22c-8809-46f3-95dc-72445c4974ab.PNG)
![sockissuedontreusebonezoomout](https://user-images.githubusercontent.com/85851448/167355857-2c5660be-7b35-420b-910b-9f4332714f51.PNG)

My running theory is that all of the bones are in "Socks" not "Arms" and "Legs" So it might be trying to work with just the meshes, but I'm not sure how to work around that.

For now I followed a different tutorial on manually assigning bones, but really want this to work! lmk if you need any more info to find a fix, thank you for taking time for us and good luck homie! <3

**Comment by Krunklehorn** [comment]
The author doesn't seem to be replying so I'll try to help instead. I'm not sure how to solve the second case, but for the former, the accessory explodes because it was exported from Blender with a different scaling setting than the avatar you're trying to apply it to. Here is a post I've made before attempting to explain the issue:

![image](https://user-images.githubusercontent.com/42424291/169710255-d90f4602-a4e4-41cb-9063-148725044542.png)

You can contact me on Discord @ Krunk#6051 if you have any more questions.

**Comment by artieficial** [comment]
Yup, scaling settings can be a nightmare, you will want to use the don't reuse bone options since it can get really screwy otherwise. Its unfortunate, but I'm not sure how to work through that yet.

**Comment by Bun** [comment]
FWIW, what seems to work is changing the Scale Factor in the Model settings of the accessory FBX. I have a set of accessories with 100x scale, so simply setting the Scale Factor to 100 on its FBX lets me reuse the avatar's bones.
